styleguide/_themes/global/scss/modifiers.scss

```diff
@@ -76,6 +76,18 @@ img { max-width: 100%; }
   margin: 0 auto;
 }
 
+// Horizontal rules
+hr {
```

Comment:
Looking through the database, we have a crap ton of `<hr>` tags, some with classes, some without.  I think we really should just create a new class for this stuff and then force some of these changes to happen on the admin side, because the implications of just styling all `<hr>` tags, seems like we're going to be perpetuating the same issues we currently have.